v0/destinations/woopra/config.js

```diff
@@ -0,0 +1,53 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://www.woopra.com/track";
+const ConfigCategories = {
+  TRACK: {
+    type: "track",
+    name: "WoopraTrackConfig"
```

Comment:
You can even add the prefix values here as well. And can import from the object for each event.

Reply:
And pass that to the common function . Yeah got it